src/lightning/__init__.py

```diff
@@ -29,9 +29,10 @@
 from lightning.pytorch.trainer import Trainer  # noqa: E402
 
 import lightning.app  # isort: skip # noqa: E402
-import lightning.store  # isort: skip # noqa: E402
+import lightning.store  # isort: skip # noqa: E402, F401
 
-lightning.app._PROJECT_ROOT = os.path.dirname(lightning.app._PROJECT_ROOT)
+_PACKAGE_ROOT = os.path.dirname(__file__)
+_PROJECT_ROOT = os.path.dirname(os.path.dirname(_PACKAGE_ROOT))
```

Comment:
Why are these 2 variables added? It doesn't seem like anything is using them?

Reply:
```suggestion
```